lib/pan_utils/include/pan_utils/pan_utils.h

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * @file pan_utils.h
+ * @author UWB Core <uwbcore@gmail.com>
+ * @date 2020
+ * @brief Personal Area Network utilities
+ *
+ */
+
+#ifndef _PAN_UTILS_H_
+#define _PAN_UTILS_H
+
+#include <stdlib.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <dpl/dpl.h>
+#include <uwb/uwb.h>
+#include <uwb/uwb_ftypes.h>
+#include <stats/stats.h>
+
+typedef enum _uwb_pan_role_t{
+    UWB_PAN_ROLE_INVALID = 0,                   //!< Pan master mode
+    UWB_PAN_ROLE_MASTER,                        //!< Pan master mode
+    UWB_PAN_ROLE_SLAVE,                         //!< Pan slave mode
+    UWB_PAN_ROLE_RELAY                          //!< Pan relay mode
+}uwb_pan_role_t;
+
+/* Copy of bootutil/image: image_version structure */
+typedef struct _pan_image_version {
+    uint8_t iv_major;
+    uint8_t iv_minor;
+    uint16_t iv_revision;
+    uint32_t iv_build_num;
+} pan_image_version;
```

Comment:
I like that you've pulled this out but I prefer it to be a struct and not a typedef. I.e. Change to: 

``` 
struct pan_image_version {
...
```

And use the struct pan_image_version in the code as per mynewt coding standard please.